content/courses/program-optimization/program-configuration.md

```diff
@@ -919,10 +911,10 @@ it("Initialize Program Config Account", async () => {
 The second test verifies that the payment instruction is working correctly, with
 the fee being sent to the fee destination and the remaining balance being
 transferred to the receiver. Here we update the existing test to include the
-`programConfig` account.
+`adminConfig` account.
 
 ```typescript
-it("Payment completes successfully", async () => {
+it("Payment should complete successfully", async () => {
```

Comment:
Grammar is off with these. 

Either:

> it("should complete successfully"

(if describe was 'payment')

or

> test("payment completes sucessfully")

See https://github.com/solana-foundation/developer-content/blob/main/CONTRIBUTING.md#jsts

Do this with all the tests.